src/app/store/slices/storage/storage.thunks/uploadItemsThunk.ts

```diff
@@ -337,51 +337,53 @@ export const uploadItemsParallelThunk = createAsyncThunk<void, UploadItemsPayloa
           uploadedFile.name = file.name;
 
           return uploadedFile;
-        })
+        }),
       );
 
       file.parentFolderId = parentFolderId;
 
       uploadPromises.map((filePromise) => {
-        return filePromise.then((uploadedFile) => {
-          const currentFolderId = storageSelectors.currentFolderId(getState());
-
-          if (uploadedFile.folderId === currentFolderId) {
-            dispatch(
-              storageActions.pushItems({
-                updateRecents: true,
-                folderIds: [currentFolderId],
-                items: uploadedFile as DriveItemData,
-              }),
-            );
-          }
+        return filePromise
+          .then((uploadedFile) => {
+            const currentFolderId = storageSelectors.currentFolderId(getState());
+
+            if (uploadedFile.folderId === currentFolderId) {
+              dispatch(
+                storageActions.pushItems({
+                  updateRecents: true,
+                  folderIds: [currentFolderId],
+                  items: uploadedFile as DriveItemData,
+                }),
+              );
+            }
 
-          tasksService.updateTask({
-            taskId: taskId,
-            merge: { status: TaskStatus.Success },
-          });
-        }).catch((err: unknown) => {
-          if (abortController.signal.aborted) {
-            return tasksService.updateTask({
-              taskId: taskId,
-              merge: { status: TaskStatus.Cancelled },
-            });
-          }
-
-          const castedError = errorService.castError(err);
-          const task = tasksService.findTask(tasksIds[index]);
-
-          if (task?.status !== TaskStatus.Cancelled) {
             tasksService.updateTask({
               taskId: taskId,
-              merge: { status: TaskStatus.Error },
+              merge: { status: TaskStatus.Success },
```

Comment:
curious why we changed this to Success if it is on a .catch 🤔

Reply:
Or is it because the diff makes it appear so? is it not on a catch?

Reply:
It looks like it has been changed because of the changes prettier has made, but in reality this lines of code
`if (task?.status !== TaskStatus.Cancelled) {`, `merge: { status: TaskStatus.Error },` are in the 376 and 379 now.

Reply:
Ah I see, makes sense